desktop/app-handlers/crash-reporting/index.js

```diff
@@ -11,10 +11,10 @@ const debug = require( 'debug' )( 'desktop:crash-reporting' );
 /**
  * Internal dependencies
  */
-const Config = require( 'lib/config' );
+const Config = require( '../../lib/config' );
 
 module.exports = function() {
-	if ( Config.crash_reporter.electron ) {
+	if ( Config.crash_reporter && Config.crash_reporter.electron ) {
```

Comment:
Also, I'll keep these in mind as once the projects are merged we'll have access to lodash's util methods here.
I think `get(Config, 'crash_reporter.electron')` might work nicely at that point.